draft-ietf-babel-information-model.md

```diff
@@ -389,61 +516,89 @@ babel-interface-metric-algorithm:
 
 babel-mcast-hello-interval:
 : The current interval in use for multicast hellos
   sent on this interface.
+  This is a 16-bit unsigned integer.
 
 babel-update-interval:
 : The current interval in use for all updates (multicast
   and unicast) sent on this interface.
+  This is a 16-bit unsigned integer.
 
 babel-message-log-enable:
-: When written, it configures whether logging should be enabled
-  (true) or disabled (false).
-  A read from the running or intended datastore indicates the
-  configured administrative value of whether logging is enabled
-  (true) or not (false). A read from the operational datastore indicates whether
-  logging is actually running (true) or not (i.e., it indicates the
-  operational state).
-  A data model that does not replicate parameters for running and operational
-  datastores can implement this as two separate parameters.
-  An implementation MAY choose
-  to expose this parameter as read-only ("ro").
+: Indicates whether message logging is enabled
+  (true) or disabled (false) on this interface.
 
 babel-message-log:
-: Log entries that have timestamp of a received Babel message
-  and the entire received Babel
-  message  (including Ethernet frame and IP headers,
-  if possible). An implementation must restrict the size of this log, but how
-  and what size is implementation-specific.
-  If this log is implemented, a mechanism
-  to clear it SHOULD be provided.
+: A reference or url link to a file that contains a timestamped log
+  of messages received and sent on babel-udp-port on this interface.
+  The {{libpcap}} file format with .pcap file extension SHOULD be supported for
+  message log files. Logging is
+  enabled / disabled by babel-message-log-enable.
+
+babel-stats-enable:
+: Indicates whether statistics collection is enabled
+  (true) or disabled (false) on this interface, including
+  neighbor-specific statistics (babel-nbr-stats).
+
+babel-stats-reset:
+: An operation that resets all babel-if-stats and babel-nbr-stats
+  parameters to zero. This
+  operation has no input or output parameters.
+
+babel-if-stats:
+: Statistics collection object for this interface.
 
 babel-neighbors:
 : A set of babel-neighbors-obj objects.
 
-babel-interface-security:
-: A babel-security-obj object that applies to this
-  interface. If implemented, this allows security to be enabled only on specific
-  interfaces or allows different security mechanisms to be enabled on different
-  interfaces.
+
+## Definition of babel-if-stats-obj
+
+~~~~
+  object {
+       uint                 ro babel-sent-mcast-hello;
+       uint                 ro babel-sent-mcast-update;
+       uint                 ro babel-received-messages;
+   } babel-if-stats-obj;
+~~~~
+{: artwork-align="left"}
+
+
+babel-sent-mcast-hello:
+: A count of the number of multicast Hello messages sent on this interface
+  since the statistics were last enabled (babel-stats-enable)
+  or reset (babel-stats-reset).

```

Comment:
Implementations handle requests for how stats are cleared and collected differently. Rather than be prescriptive on what the value of stats represent, I would leave the description as "A count of the number of multicast Hello messages sent on this interface."

This relates to my previous comment on enabling/disabling stats.

Reply:
OK